pyrad/tests/testPacket.py

```diff
@@ -186,6 +231,42 @@ def testVerifyReply(self):
         self.assertEqual(self.packet.VerifyReply(reply), False)
         reply.authenticator = self.packet.authenticator
 
+    def testVerifyReplyDuplicateAttributes(self):
+        reply = self._create_reply_with_duplicate_attributes(self.packet)
+        self.assertTrue(self.packet.VerifyReply(
+            reply=reply,
+            rawreply=reply.raw_packet))
```

Comment:
Added a test to cover the fix for #140 by commit https://github.com/pyradius/pyrad/commit/2c9a455784a636148400d79885e8819169032831 while I was at it.